commit 2f5a6b906ad86ef6570863a75b204551c2c62fec
Author: P33M <p33m@github.com>
Date:   Thu May 2 11:53:45 2019 +0100

    lan78xx: use default alignment for rx buffers
    
    The lan78xx uses a 12-byte hardware rx header, so there is no need
    to allocate SKBs with NET_IP_ALIGN set. Removes alignment faults
    in both dwc_otg and in ipv6 processing.

diff --git a/drivers/net/usb/lan78xx.c b/drivers/net/usb/lan78xx.c
index 3387f44b132f..14f206a56187 100644
--- a/drivers/net/usb/lan78xx.c
+++ b/drivers/net/usb/lan78xx.c
@@ -3250,7 +3250,7 @@ static int rx_submit(struct lan78xx_net *dev, struct urb *urb, gfp_t flags)
 	size_t size = dev->rx_urb_size;
 	int ret = 0;
 
-	skb = netdev_alloc_skb_ip_align(dev->net, size);
+	skb = netdev_alloc_skb(dev->net, size);
 	if (!skb) {
 		usb_free_urb(urb);
 		return -ENOMEM;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1486_2f5a6b906ad86ef6570863a75b204551c2c62fec.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1486_2f5a6b906ad86ef6570863a75b204551c2c62fec.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
